Configuration/StandardSequences/python/RawToDigi_cff.py

```diff
@@ -74,6 +76,8 @@
 muonDTDigis.inputLabel = 'rawDataCollector'
 muonRPCDigis.InputLabel = 'rawDataCollector'
 castorDigis.InputLabel = 'rawDataCollector'
+totemTriggerRawToDigi.rawDataTag = cms.InputTag("rawDataCollector")
+totemRPRawToDigi.rawDataTag = cms.InputTag("rawDataCollector")
```

Comment:
can we have the CTPPS unpackers instances follow the same naming style as all other unpackers?
More specifically, totemRPDigis instead of totemRPRawToDigi (the totemTriggerRawToDigi eventually too, but it's more internal now).

Considering that it's now in so many pending PRs, it makes more sense to migrate later.